integration_test/metadata/integration_metadata.go

```diff
@@ -155,7 +155,7 @@ func UnmarshalAndValidate(fullPath string, data []byte, i interface{}) error {
 	v := NewIntegrationMetadataValidator()
 	// Note: Unmarshaler does not protect when only the key being declared.
 	// https://github.com/goccy/go-yaml/issues/313
```

Comment:
Please update this comment if this issue (# 313) has been fixed with the current update.

Reply:
Won't be fixed until 1.14.3, which is available if we want to update to that.